apps/src/applab/ViewDataButton.jsx

```diff
@@ -0,0 +1,49 @@
+/** @file Button that opens the data browser */
+// Strict linting: Absorb into global config when possible
+/* jshint
+ unused: true,
+ eqeqeq: true,
+ maxlen: 120
+ */
+/* global $ */
+
+var applabMsg = require('./locale');
+var styles = require('./PlaySpaceHeaderStyles');
+
+/**
+ * The button above the visualization that opens the data browser.
+ * @type {function}
+ */
+var ViewDataButton = React.createClass({
+  propTypes: {
+    onClick: React.PropTypes.func.isRequired
+  },
+
+  render: function () {
+
```

Comment:
I've been seeing various folks stick an empty line at the top of functions in a bunch of places recently. Any reason for this? Not a big deal really, just wondering if there's a justification. I personally don't like it, but can live with it :-P